.changeset/slimy-avocados-thank.md

```diff
@@ -0,0 +1,6 @@
+---
+"docs": patch
```

Comment:
@rociobaena let's remove `docs` from this changeset.

Reply:
right!

Reply:
Removed

Reply:
@rociobaena docs is still part of this changeset.

Please create a ticket for making a Danger rule that will check to make sure docs are not part of a Changeset, as I think this will help any new team members in the future, and we have seen this pop up a few times.